packages/wallet-sdk/src/components/RedirectDialog/RedirectDialog.scss

```diff
@@ -0,0 +1,53 @@
+@import "../theme.scss";
```

Comment:
we should add dark mode support

Reply:
what would you recommend for dark mode? do you think background and font color would be enough?

btw, note that this method is supposed to be used by AMEX only first, and don't have plan to expose this externally.

Reply:
Gotcha probably fine to leave for now

Reply:
we can re visit coloring later (this project is for a hack so it doesn't have design support), but here's darkmode @cb-jake.
note that this dialog is only for an edge case where the browser blocks redirection.
![Simulator Screenshot - iPhone 13 mini - 2023-06-29 at 13 23 45](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/05e8c887-8a1a-4e5c-b011-dbcb7c5fb443)
![Simulator Screenshot - iPhone 13 mini - 2023-06-29 at 13 20 25](https://github.com/coinbase/coinbase-wallet-sdk/assets/966662/362463a5-deaf-42b3-9207-75443a33a301)